community/neo4j/src/test/java/org/neo4j/index/backup/IndexBackupIT.java

```diff
@@ -249,4 +253,9 @@ private DependencyResolver getDatabaseResolver()
     {
         return database.getDependencyResolver();
     }
+
+    private boolean segmentsFilePredicate( String fileName )
+    {
+        return fileName.contains( "segments_" );
```

Comment:
Since we compare absolute paths here I think it's safer to use
```
return new File(fileName).getName().startsWith( org.apache.lucene.index.IndexFileNames.SEGMENTS );
```
and avoid false positives.